internal/distro/fedora/package_sets.go

```diff
@@ -436,6 +436,47 @@ func iotInstallerPackageSet(t *imageType) rpmmd.PackageSet {
 	return ps
 }
 
+func liveInstallerPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@workstation-product-environment", // in ks this is @^, what does that mean?
+			"@anaconda-tools",
+			"anaconda-install-env-deps",
+			"anaconda-live",
+			"anaconda-dracut",
+			"dracut-live",
+			"glibc-all-langpacks",
+			"kernel",
+			"kernel-modules",
+			"kernel-modules-extra",
+			"livesys-scripts",
+			"rng-tools", // XXX these should be included right!?
```

Comment:
nitpick related to the comment: when we will know the answer? It looks like a WIP comment, which does not provide much value once we merge it, does it?

Reply:
The comment isn't relevant anymore, removed :)